app/assets/src/components/visualizations/TidyTree.js

```diff
@@ -209,6 +214,7 @@ export default class TidyTree {
         y0: node.y
       })
     );
+    this.options.onCollapsedStateChange(node);
```

Comment:
Check if the function exists first: 
`this.options.onCollapsedStateChange && this.options.onCollapsedStateChange(node)`